galsim/utilities.py

```diff
@@ -418,3 +418,102 @@ def _gammafn(x):
                0.00000000000000000141, -0.00000000000000000023, 0.00000000000000000002
              )
 
+def interleaveImages(im_list,N,offsets=None,add_flux=True,suppress_warnings=False):
+    """
+    Interleaves two or more images and outputs a larger image.
+
+    The sampling length of simulated images can be set arbitrarily using the `pixel_scale' argument
+    in drawImage() routine appropriately. However, pixel level detector effects can be included
+    only on images drawn at the native pixel scale, which are typically undersampled. Nyquist
+    sampled images that also include the effects of detector non-idealities can be obtained by
+    drawing multiple undersampled images (with the detector effects included) that are offset from
+    each other by a fraction of a pixel. This is equivalent to obtaining a finer sampled image from
+    a dither sequence, except that this routine handles only equispaced offsets. The dither sequence
+    must be a list of galsim.Image instances supplied through 'im_list'.
+
+    @param im_list           A list containing the galsim.Image instances to be interleaved.
+    @param N                 Number of images to interleave in either direction. It can be of type
+                             `int' if equal numbers of images are interleaved in both directions or
+                             a tuple of two integers, containing the number of images in x and y
+                             directions respectively.
+    @param offsets           A list containing the offsets as galsim.PositionD instances
+                             corresponding to every image in `im_list'. The offsets must be equally
+                             spaced and must span an entire pixel area. The offset values must
+                             be symmetric around zero, hence taking positive and negative values.
+                             The default offset ordering is to vary the offset in x from positive to
```

Comment:
I would have naively expected the offsets to go from negative to positive.  I agree with @rmandelb , probably best to just require the offsets explicitly.